runtime/exports.t4

```diff
@@ -177,23 +177,23 @@
 
 		#region metadata/profiler.h
 
-		new Export ("void", "mono_profiler_set_events",
-			"MonoProfileFlags", "events"
+		new Export ("MonoProfilerHandle", "mono_profiler_install",
```

Comment:
OK, and just to confirm, it's fine that if someone wants to profile, we tell them to upgrade their system Mono, right?